docs/content/en/docs/workshops/provision/vSphere-prepeartion.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Could you change the filename from "vSphere-prepeartion.md" to  "vSphere-preparation.md"?